### DISCUSSION TITLE
Combine two images together

### DISCUSSION THREAD

**Discussion by jaroslavhuss** [lllyasviel/Fooocus#1803] [Q&A] [answered]
Dear Community, 

is it even possible (I was searching for so long) to combine two images like this?: 

1st image: 
![2024-01-08_07-07-10_4175](https://github.com/lllyasviel/Fooocus/assets/22031248/29a44ac6-a30e-4243-85fa-14f2192b5e11)
and 2nd image:

![GFF box 2019 new](https://github.com/lllyasviel/Fooocus/assets/22031248/4c2db6e0-b518-4295-82e3-e8721c206d51)

Basically, I need those girls to hold the GlobiFer package. I could not find any input image combination that would just take the girls and let them hold the package... Maybe there is no such way or is it? 

Thank you for any help, guidance.

**Comment by thiner** [comment]
I don't think Fooocus can change object precisely. It takes all image inputs as PROMPT.

**Comment by jaroslavhuss** [comment]
Ok, thank you! at least I am about to stop trying :-))))

**Comment by KitsonBroadhurst** [comment, marked as answer]
@jaroslavhuss I'd probably attempt it in two ways:
- combining the advanced image prompting with a descriptive text prompt. Add the first image with three women, from the advanced image prompt options choose PyraCanny. Add the second image of the product as a normal image prompt. Finally write a descriptive text prompt
- combining  an image prompt with inpainting, that will leave the original base image and attempt to change the "product". This video should show you the advanced checkbox for combining image prompt and inpainting: https://www.youtube.com/watch?v=nWvOzBG1k_k

**Comment by tresorama** [comment]
Did you find a solution ?

**Comment by sandrapisano** [comment]
how to generate two characters together on Fooocus colab precise a man and a woman hugging but from separate portraits ask him to put them hugging in a photo ?
